spec.html

```diff
@@ -32043,92 +32043,269 @@ <h1>Time Values and Time Range</h1>
         <p>A Number can exactly represent all integers from -9,007,199,254,740,992 to 9,007,199,254,740,992 (<emu-xref href="#sec-number.min_safe_integer"></emu-xref> and <emu-xref href="#sec-number.max_safe_integer"></emu-xref>). A time value supports a slightly smaller range of -8,640,000,000,000,000 to 8,640,000,000,000,000 milliseconds. This yields a supported time value range of exactly -100,000,000 days to 100,000,000 days relative to midnight at the beginning of 1 January 1970 UTC.</p>
         <p>The exact moment of midnight at the beginning of 1 January 1970 UTC is represented by the time value *+0*<sub>𝔽</sub>.</p>
         <emu-note>
+          <p>In the proleptic Gregorian calendar, leap years are precisely those which are (divisible by 4) and ((not divisible by 100) or (divisible by 400)).</p>
```

Comment:
This should be unambiguous without introducing disambiguating parens in prose.

```suggestion
          <p>In the proleptic Gregorian calendar, leap years are precisely those which are both divisible by 4 and either divisible by 400 or not divisible by 100.</p>
```

Reply:
Note that this is pre-existing text, just moved (from the dissolved "Year Number" section), so not this PR's problem. But I made the change anyway.

Personally, I like formulations in which the divisors are in order (ascending or descending). E.g.:
- a year is a common year,
- unless it's divisible by 4, in which case it's a leap year,
- unless it's divisible by 100, in which case it's a common year,
- unless it's divisible by 400, in which case it's a leap year.

or:
- If it's divisible by 400, it's a leap year.
- Otherwise, if it's divisible by 100, it's a common year.
- Otherwise, if it's divisible by 4, it's a leap year.
- Otherwise, it's a common year.

Reply:
Those seem like they'd be fine, too.